content/news/2025-01-22_rocur-relaunch/index.en.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "We're Launching the R-Ladies Rotating Curation (RoCur) Account on Bluesky!"
+author: "RoCur team"
+type: blog
+date: "2025-01-22"
+image: img/rladies-bluesky.jpg
+tags: 
+- rocur
+categories: 
+- R-Ladies
+summary: We're looking for curators for the We Are R-Ladies Bluesky account!
+slug: rocur-relaunch-2025
+---
+
+![Screenshot of We Are R-Ladies on Bluesky](img/rladies-bluesky.jpg)
+
+We are absolutely thrilled to announce the R-Ladies Rotating Curation (RoCur) Bluesky account [bsky.app/profile/weare.rladies.org](https://bsky.app/profile/weare.rladies.org) is now live! Each week, a new member of the R-Ladies community will take over the account to talk about all things R. This will be a fantastic opportunity for individuals who identify as a minority gender in the R community to amplify their voices and connect with others in the greater data science community.
+
+## History
+
+Some of you might recall the RoCur account on Twitter. The program ran for about five years, giving over 100 curators the chance to take over the account and share their stories. While the Twitter account is no longer active, the archive of posts can still be viewed [here](https://rladies.org/tweet-archive/wearerladies/).
+
+## How You Can Help
+
+We are excited to revive the RoCur program on Bluesky, picking up where we left off. You can help us build back the community stronger than ever by signing up to be a curator or nominating someone to be a curator!
+
+## Become a Curator
+
+*Are you interested in becoming a curator?* Curating is a great way to share and learn new R techniques, build your social media presence, develop your science communication skills, and connect with a supportive community. We welcome curators of all experiences levels and professional backgrounds. Curation may be conducted in any language; posts are not restricted to English!
```

Comment:
Just a note, if someone starts curating in say French, will the default language on posts be changed? To make it less likely they post without specifying it is French? (if the post is correctly classified in a language, then people can click on "translate")

Reply:
Thanks for the tip! If a potential curator reaches out about posting in a language other than English, we'll make sure to update the [FAQ page](https://guide.rladies.org/rocur/faq/) to show how you can specify the language(s) that's used in the post.